examples/react-apollo/cypress/support/e2e.ts

```diff
@@ -22,10 +42,58 @@ Cypress.Commands.add('signUpEmailPassword', (email, password) => {
   cy.contains('Continue with email + password').click()
 })
 
+Cypress.Commands.add('signUpEmailPasswordless', (email) => {
+  cy.visit('/sign-up')
+  cy.contains('Continue with passwordless email').click()
```

Comment:
`cypress-testing-library` exposes a helper, which may be useful here: `findByRole`.

Instead of `cy.contains('...')` we could use

```js
cy.findByRole('button', { name: /continue with passwordless email/i })
```

We can make the code more declarative here and also utilize cypress-testing-library even better.

For more information, see: https://testing-library.com/docs/cypress-testing-library/intro/#examples